main.go

```diff
@@ -140,21 +124,43 @@ func main() {
 				"GCP-ZONE":    zone,
 				"INSTANCE-IP": ip,
 			}
+		default:
+			fmt.Fprintf(os.Stderr, "Error: unknown deployment type \n")
```

Comment:
`dType` looks to be missing in the error string

Reply:
`getDeploymentType()` returns a literal `deploymentTypeUnknown` for cases where the request to the metadata server errored or is the response from MDS contains an unknown "Server" value. 

a better way would be perhaps to return the error from `getDeploymentType()` and put that in the error message. I can do that as a follow up

Reply:
Okay. Well, the error isn't at all clear what happened or what the user should do. And I thought if you added more details you would have noticed the trailing space before the newline.

Reply:
https://github.com/GoogleCloudPlatform/traffic-director-grpc-bootstrap/pull/51 in flight to address this